### PR TITLE
Update the SDK version #431

### PR DESCRIPTION
**Pull request by phlniji**
#431

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
       # Make sure we use the latest version of MatrixKit
       - name: Reset MatrixKit pod
         run: rm -rf Pods/MatrixKit
-
+
```
Comment (giomfo) [inline .github/workflows/ci-build.yml:42]: I think there is a mistake here 
we should remove MatrixKit pod and keep MatrixSDK pod (like in element-ios [here](https://github.com/vector-im/element-ios/blob/develop/.github/workflows/ci-build.yml#L40)
Don't you think so?

File: .github/workflows/ci-tests.yml [hunk 2]
```diff
@@ -39,10 +39,6 @@ jobs:
       # Make sure we use the latest version of MatrixKit
       - name: Reset MatrixKit pod
         run: rm -rf Pods/MatrixKit
-        
-      # Make sure we use the latest version of MatrixSDK
-      - name: Reset MatrixSDK pod
-        run: rm -rf Pods/MatrixSDK
```
Comment (giomfo) [inline .github/workflows/ci-tests.yml]: idem - same comment

File: .github/workflows/ci-build.yml (hunk 1 above)
Comment (phlniji) [reply .github/workflows/ci-build.yml:42]: Changes here are just to clean the cache for CI.
The reset MatrixKit will be replaced by reset MatrixSDK with the next rebase.

File: .github/workflows/ci-tests.yml (hunk 2 above)
Comment (phlniji) [reply .github/workflows/ci-tests.yml]: Changes here are just to clean the cache for CI.
The reset MatrixKit will be replaced by reset MatrixSDK with the next rebase.

**Review by giomfo — APPROVED** [review]

**Pull request merged**